book/aave_proposals.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
Notebooks are getting updated via the Github action only on changes to `main`.

Hence, it's important to show to the reader the date at which the notebook has been run, to know how fresh is the data.

I suggest we add the date right below the title of all notebooks.